eng/pipelines/dotnet-buildtools-prereqs-alpine.yml

```diff
@@ -25,3 +25,10 @@ stages:
   parameters:
     internalProjectName: ${{ variables.internalProjectName }}
     publicProjectName: ${{ variables.publicProjectName }}
+    ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+      customCopyBaseImagesInitSteps:
+      - template: ../../../pipelines/steps/set-base-image-override-options.yml
+        parameters:
+          variableName: customCopyBaseImagesArgs
+          dockerfileOs: alpine
+          baseOverrideRegistry: $(overrideRegistry) # Comes from DotNet-Docker-Common variable group
```

Comment:
Has there been assessment of the override registry to ensure it has available all the image tags that these Dockerfiles require?

Reply:
I skipped some of the OSes specifically because the override registry doesn't have them, but didn't look specifically into OS versions. I'll do another pass and make sure.

Reply:
I have confirmed that all the cases that we would be replacing have the image tags in the registry